test/ffmpeg-qsv/transcode/avc.py

```diff
@@ -0,0 +1,190 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .transcoder import TranscoderTest
+
+spec = load_test_spec("avc", "transcode")
+class to_avc(TranscoderTest):
+  @slash.requires(have_ffmpeg_h264_qsv_encode)
+  @slash.requires(have_ffmpeg_h264_qsv_decode)
```

Comment:
Please put the decode requirement checks before the encode requirement checks.  It will make it logically easier to read the code since transcode implies decode->encode.

Reply:
changed the order of functions